getting_started/step_by_step/scene_tree.rst

```diff
@@ -140,10 +146,18 @@ another one. The simple way to do this is to use the
 :ref:`SceneTree.change_scene() <class_SceneTree_change_scene>`
 function:
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     func _my_level_was_completed():
-        get_tree().change_scene("res://levels/level2.tscn")
+    get_tree().change_scene("res://levels/level2.tscn")
```

Comment:
Probably we should fix formatting here, since I heard that GDScript is similar to Python which has a strict indentation rule.